test/controllers/PaperlessPreferencesControllerSpec.scala

```diff
@@ -74,8 +63,7 @@ class PaperlessPreferencesControllerSpec extends BaseSpec with MockitoSugar {
         })
 
         val r = controller.managePreferences(FakeRequest())
-        status(r) shouldBe OK
-        verify(controller.preferencesFrontendPartialService, times(1)).getManagePreferencesPartial(any(), any())(any())
+        status(r) shouldBe SEE_OTHER
```

Comment:
Assert the redirect URL, either by mocking Tools or asserting the start of the Url